apidef/oas/schema/x-tyk-gateway.md

```diff
@@ -42,16 +44,83 @@ Tyk classic API definition: `name`.
 Expiration date.
 
 **Field: `state` ([State](#state))**
-State holds configuration about API definition states (active, internal).
+State holds configuration for API definition states (active, internal).
 
 **Field: `versioning` ([Versioning](#versioning))**
 Versioning holds configuration for API versioning.
 
+# Upstream
+
+Upstream holds configuration for the upstream server to which Tyk should proxy requests.
+
+**Field: `url` (`string`)**
+URL defines the upstream address (or Target URL) to which requests should be proxied.
+
+Tyk classic API definition: `proxy.target_url`.
+
+**Field: `serviceDiscovery` ([ServiceDiscovery](#servicediscovery))**
+ServiceDiscovery contains the configuration related to Service Discovery.
+
+Tyk classic API definition: `proxy.service_discovery`.
+
+**Field: `test` ([Test](#test))**
+Test contains the configuration related to uptime tests.
+
+**Field: `mutualTLS` ([MutualTLS](#mutualtls))**
+MutualTLS contains the configuration for establishment of mutual TLS between Tyk and the upstream server.
+
+**Field: `certificatePinning` ([CertificatePinning](#certificatepinning))**
+CertificatePinning contains the configuration related to certificate pinning.
+
+# Server
+
+Server contains the configuration that sets Tyk up to receive requests from the client applications.
+
+**Field: `listenPath` ([ListenPath](#listenpath))**
+ListenPath is the base path on Tyk to which requests for this API should
+be sent. Tyk listens for any requests coming into the host at this
+path, on the port that Tyk is configured to run on, and processes these
+accordingly.
+
+**Field: `slug` (`string`)**
+Slug is the Tyk Cloud equivalent of listen path.
+
+Tyk classic API definition: `slug`.
+
+**Field: `authentication` ([Authentication](#authentication))**
+Authentication contains the configurations that manage how clients can authenticate with Tyk to access the API.
+
+**Field: `clientCertificates` ([ClientCertificates](#clientcertificates))**
+ClientCertificates contains the configurations related to establishing static mutual TLS beetween the client and Tyk.
+
+**Field: `gatewayTags` ([GatewayTags](#gatewaytags))**
+GatewayTags contains segment tags to configure which GWs your APIs connect to.
```

Comment:
I'd prefer (for consistency) to avoid the GW abbreviation here so could we please instead have:

```suggestion
GatewayTags contains segment tags to indicate which Gateways your upstream service is connected to (and hence where to deploy the API).
```